package.json

```
@@ -8,7 +9,9 @@
     "tsc": "tsc --noEmit",
     "test": "jest",
     "prepare": "bob build",
-    "semantic-release": "semantic-release"
+    "semantic-release": "semantic-release",
+    "docs:start": "yarn workspace docs run start",
```

AFAIK, You don't need `run` on `yarn`.
Also, I'm thinking if we need 2 scripts for that, maybe just aliasing workspace is enough.

```suggestion
    "docs": "yarn --cwd docs",
```

---

`yarn --cwd docs` doesn't work for that (just tested id), if we want to use `yarn docs start` we must use `"docs": "yarn workspace docs"`

---

Fixed with `"yarn workspace docs"`! thanks